packages/ButtonGroup/src/ButtonGroup.js

```diff
@@ -90,6 +93,24 @@ function ButtonGroup(props) {
     setFocusIndex(index);
   }
 
+  React.useLayoutEffect(() => {
+    const $label = refLabel && refLabel.current;
+
+    if (!$label) return;
+
+    function handleClickLabel() {
+      const enabledIndexes = getEnabledIndexes();
+      focusButton(enabledIndexes[0]);
+      console.log("focusing first button", document.activeElement);
```

Comment:
currently finding the button focusing but the imperative api `focus` function isn't being triggered. Investigating whether a dedicated ref is needed for each `buttonGroup.Item` in order the call the `focus()` api on that component.